src/WebView.android.tsx

```diff
@@ -76,6 +76,7 @@ class WebView extends React.Component<AndroidWebViewProps, State> {
     lastErrorEvent: null,
   };
 
+  onShouldStartLoadWithRequest: ReturnType<typeof createOnShouldStartLoadWithRequest> | null = null;
```

Comment:
Perhaps rename things in this file to make it more clear to the reader that this is distinct? It's kind of confusing the way things are written now.

Reply:
I'll play with it and see how much better I can make it.